.lintstagedrc

```diff
@@ -0,0 +1,7 @@
+{
+	"scripts/**/*": "prettier --write",
+	"packs_source/**/*": "node tooling/prettier/prettier-packs.js",
```

Comment:
Added a custom wrapper for files in `packs_source`. They get the standard `prettier` treatment, but then an additional post-processing step to re-format some lines.